tests/control/PlanarArmControllerTest.cpp

```diff
@@ -9,5 +11,23 @@ using namespace std::chrono_literals;
 TEST_CASE("Test Planar Arm Controller", "[control][planararmcontroller]") {
 	navtypes::Vectord<2> vec({0, 0});
 	kinematics::PlanarArmKinematics<2> kin_obj(vec, vec, vec, 0.0, 0);
-	PlanarArmController<2> foo({0, 0}, kin_obj);
+	PlanarArmController<2> foo({0, 0}, kin_obj, Constants::arm::SAFETY_FACTOR);
+}
+
+TEST_CASE("Test Planar Arm Safety Factor", "[control][planararmcontroller]") {
+	navtypes::Vectord<2> segLens({4.0, 6.0});
+	navtypes::Vectord<2> minAngles({-M_PI, -M_PI});
+	navtypes::Vectord<2> maxAngles({M_PI, M_PI});
+	kinematics::PlanarArmKinematics<2> kin_obj(segLens, minAngles, maxAngles, 0.0, 0);
+
+	PlanarArmController<2> foo({0, 0}, kin_obj, Constants::arm::SAFETY_FACTOR);
+	foo.set_setpoint({10.0, 0.0});
+	REQUIRE(foo.get_setpoint(robot::types::dataclock::now()).norm() == 9.5);
+
+	foo.set_setpoint({2.0, 3.0});
+	Eigen::Vector2d origSetPoint = kin_obj.jointPosToEEPos({2.0, 3.0});
+	double expectedNorm = origSetPoint.norm();
+
+	REQUIRE(std::abs(foo.get_setpoint(robot::types::dataclock::now()).norm() - expectedNorm) <
+			1e-6);
```

Comment:
Instead of checking that the norms are equal, can we check for vector equality? Some of the other tests do the same thing, so you can copy-paste that code